tests/Extensions/Hosting.Services.Remoting.UnitTests/GenericRemotingListenerBuilderTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.Omex.Extensions.Hosting.Services.UnitTests;
+using Microsoft.ServiceFabric.Services.Remoting;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services.Remoting.UnitTests
+{
+	[TestClass]
```

Comment:
nit: TestCategory :)

Reply:
But all of those tests a for the same project what would be the proper category?

Reply:
I'll leave it without category, for now, we can discuss out test category approach later